de4a-connector/src/main/java/eu/de4a/connector/error/handler/ConnectorExceptionHandler.java

```diff
@@ -23,6 +23,6 @@ public String getMessage(ConnectorException ex) {
         }
     }
 
-    public abstract Object getResponseError(ConnectorException exception, boolean returnString);
+    public abstract Object getResponseError(ConnectorException exception, boolean returnBytes);
```

Comment:
to overloads - one for bytes and one for T seem to be smarter... Or at least add default methods in here, that do the casting